### ISSUE TITLE
Changes to Wargames List

### ISSUE THREAD

**Issue by IanMayo** [serge-web/serge#551]
Celine has provided some feedback on the Organisms/Wargames-List/Lots-of-wargames Storybook component.

![image](https://user-images.githubusercontent.com/1108513/97161432-baf28300-1775-11eb-9115-d7de4f6d663a.png)

I asked:
```
The “grab” (thumb) part of the scrollbar is black. It appears on hover. But, I don’t think there’s a
clear indication that there is more than a screenful of items in the list.
My gut feeling is that the “grab” part should be permanently visible, and that it should be in a 
color that offers more contrast.

Do you have any thoughts?
```

Caline replied:
```
The grab bar should be a light grey. but most of all, the lines that separate the games should not overlap with the bar.

There is no need to have the grey slider background on hover. The grab bar can just turn white.

Also the war game menu is almost covered by the scroll bar, as it appears only on hover, it could be next 
to the Wargame name. Right now it is easy to miss, especially on a large screen.

For the last visible game on the list, the menu is not entirely visible
```
![image](https://user-images.githubusercontent.com/1108513/97161634-00af4b80-1776-11eb-966d-e24a7d09dd3b.png)

- [x] Grab bar should be light grey
- [x] Lines that separate the games should not overlap with the bar
- [x] No need to have the grey slider background on hover. The gab bar can just turn white
- [x] Game menu almost covered by the scroll bar, as it appears only on hover.   Make it appear next to the wargame name
- [x] Ensure menu is fully visible for the last game on the list

**Comment by IanMayo** [comment]
@sanjayradadiya  said:
```
Can you give me some idea about this point ? actually where menu needs to be display
Game menu almost covered by the scroll bar, as it appears only on hover. Make it appear next to the wargame name
```

Ok. The comment refers to the popup menu that appears when you click the three dots.
![image](https://user-images.githubusercontent.com/1108513/114306743-bed05280-9ad4-11eb-8d95-c719456fa378.png)

Celine was requesting that three-dot  component be moved next to the wargame title.  So, instead of being at the right-hand side of the list item, it's adjacent to the text label.

**Comment by IanMayo** [comment]
Close. Fixed in #663
